package-lock.json

```diff
@@ -1,133 +1,8 @@
 {
     "name": "gmod-typescript",
     "version": "1.0.0",
-    "lockfileVersion": 2,
```

Comment:
Please update your npm/node version and rerun npm install. That way, the lock file version won't get downgraded.